utils/dataloaders.py

```diff
@@ -991,6 +996,7 @@ def verify_image_label(args):
     # Verify one image-label pair
     im_file, lb_file, prefix = args
     nm, nf, ne, nc, msg, segments = 0, 0, 0, 0, '', []  # number (missing, found, empty, corrupt), message, segments
+    is_format_tagged = True
```

Comment:
Why the default value is True?

Reply:
the truth value is adjusted later based on what is in the txt file, but I can make this a bit clearer indeed